ProcessLib/RichardsFlow/RichardsFlowFEM.h

```diff
@@ -96,6 +108,11 @@ class LocalAssemblerData : public RichardsFlowLocalAssemblerInterface
             _integration_method.getNumberOfPoints();
         SpatialPosition pos;
         pos.setElementID(_element.getID());
+
```

Comment:
With this implementation, the velocity is always from the previous time step.

For the correct calculation, you need to
1)  put the calculation of the velocity by overloading this member of LocalAssember
https://github.com/ufz/ogs/blob/master/ProcessLib/LocalAssemblerInterface.h#L69
2)  overload the Process:computeSecondaryVariableConcrete